ckanext/geocat/tests/test_harvester.py

```diff
@@ -1,256 +1,103 @@
-import json
 import logging
 import os
 
-import ckan.plugins.toolkit as tk
 import ckantoolkit.tests.helpers as h
 import pytest
-import requests
-import requests_mock
-from ckan import model
-from ckan.common import config
 
-from ckanext.harvest import queue
+from ckanext.geocat.harvester import GeocatHarvester
+from ckanext.harvest.tests.lib import run_harvest
 
 log = logging.getLogger(__name__)
 
 __location__ = os.path.realpath(os.path.join(os.getcwd(), os.path.dirname(__file__)))
 
 MOCK_URL = "http://mock-geocat.ch"
 mock_record_url = "http://mock-geocat.ch/geonetwork/srv/eng/csw-BAKOM"
-mock_capabilities_url = (
-    "http://mock-geocat.ch/?version=2.0.2&request=GetCapabilities&service=CSW"
-)
-clear_solr_url = (
-    config.get("solr_url")
-    + "/update?stream.body=%3Cdelete%3E%3Cquery%3Ename:geocat-harvester%20OR%20organization:geocat_org%3C/query%3E%3C/delete%3E&commit=true"
-)
```

Comment:
We no longer need to clear Solr like this, as we can use the pytest-ckan fixture `clean_index`.